### PR TITLE
A Database Secrets Engine plugin for Aerospike

### PR DESCRIPTION
**Pull request by rbotzer**

### CODE REVIEW

**Review by acahn — APPROVED** [review]
Looks good. Thank you for being a HashiCorp Partner!

File: website/pages/docs/plugin-portal/index.mdx [hunk 1]
```diff
@@ -101,6 +101,10 @@ reviewed by HashiCorp.
 
 - [Venafi](https://github.com/Venafi/vault-pki-backend-venafi)
 
+### Database
+
+- [Aerospike](https://github.com/aerospike-community/vault-plugin-database-aerospike)
+
```
Comment (calvn) [inline website/pages/docs/plugin-portal/index.mdx:107]: Can this be moved up to be above `### Secrets` to keep it consistent with the "Official" layout.

Comment (acahn) [reply website/pages/docs/plugin-portal/index.mdx:107]: Yes.  That makes sense.

Comment (rbotzer) [reply website/pages/docs/plugin-portal/index.mdx:107]: I can swap the sub-sections to keep them in consistent order.

Comment (rbotzer) [reply website/pages/docs/plugin-portal/index.mdx:103]: @calvn @acahn Done.

Comment (acahn) [reply website/pages/docs/plugin-portal/index.mdx:103]: LGTM!  Thank you Ronen

**Pull request merged**